.github/workflows/publish.yaml

```diff
@@ -47,6 +47,10 @@ jobs:
             packages/*/dist
             packages/*/build
           key: build-cache-${{ github.run_id }}-${{ github.run_attempt }}
+      - name: Authenticate with private GH package
+        run: echo -e "//npm.pkg.github.com/:_authToken=${{ steps.vault.outputs.GITHUB_PACKAGES_WRITE_TOKEN }}\n@contentful:registry=https://npm.pkg.github.com" > ./.npmrc
```

Comment:
Would it be better to be more explicit about the file path and use `~/.npmrc` over a relative path

Reply:
This puts it in the current directory. Originally it was the home directory like you mentioned, but I think I changed to to the local directory trying to get it to work. Either place should work.